packages/styles/scss/globals/utils/_ratio-calc.scss

```diff
@@ -0,0 +1,10 @@
+// Ratio calculation
+// function to reuse the ratio-calc
+// @param $width
+//   width of the element
+// @param $height
+//   height of the element
+// @return percentage of height vs width
+@function ratio-calc($width, $height) {
+  @return percentage($height / $width);
```

Comment:
I think you copied my example one for one when I was trying to demonstrate some of the logic rather than focusing on being syntactically correct. I don't think you're using the right function here.

https://sass-lang.com/documentation/modules/math